.coveragerc

```diff
@@ -1,7 +1,25 @@
 [run]
+source_pkgs =
+	setuptools
+	pkg_resources
 omit =
 	# leading `*/` for pytest-dev/pytest-cov#456
-	*/.tox/*
+	*/_vendor/*
+	*/_distutils/*
+	*/tests/*
+	*/test_*
+
+[paths]
+setuptools =
+	setuptools/
+	*/site-packages/setuptools/
+	*/pip-install-*/setuptools_*/setuptools/
+	*/tmp_*/setuptools
+pkg_resources =
+	pkg_resources/
+	*/site-packages/pkg_resources/
+	*/pip-install-*/setuptools_*/pkg_resources/
+	*/tmp_*/pkg_resources
```

Comment:
The combination of the project layout and the kinds of tests we run is rather unique (2 top-level packages, in-tree tests and flat-layout, tests using pip-run), which require a bespoke path configuration...

For most of packages using src-layout, the following configuration is enough:
```ini
[paths]
source =
	src/
	*/site-packages/
```
The equivalent for flat-layout is:
```ini
[paths]
source =
	../  # seems to be resolved against the `.coveragerc` file path, not the current folder
	*/site-packages/
```

PS: `coverage` does not allow specifying wildcards as the last segment in `[paths]`, therefore we need an entry for each `setuptools` and `pkg_resources`.
`*/pip-install-*` paths seem to be created by `pip-run` and `*/tmp_*` paths are created when installing into directories created with the `tempfile` module.